schedule/reminders.js

```diff
@@ -115,18 +115,19 @@ module.exports = {
         var db = options.db;
 
         // gets all clinics
-        db.medic.view('medic', 'clinic_by_phone', {
+        db.medic.view('medic-client', 'doc_by_type', {
```

Comment:
that's just unrelated! Lucky for you that @alxndrsn is not reviewing this, cos it would NOT FLY 😛